Lombiq.TrainingDemo.Web/wwwroot/.placeholder

```diff
@@ -0,0 +1 @@
+��
```

Comment:
I removed the strange character.

This file is created by OC's codegen template. I remember that without this, we had trouble with the app creating files under the wwwroot when deployed (probably it won't even try to create the folder but just fail to write files). It's perhaps not needed anymore, but I take keeping it as an artifact of dogmatic faith over having to do a cross-platform investigation of whether it can be removed :). Hence I wouldn't touch it in OC either (I tried this and locally it seems fine, as does under the Ubuntu CI build, so this seems safe).

Reply:
![image](https://github.com/Lombiq/Orchard-Training-Demo-Module/assets/4007293/61a47344-12a4-4074-b8ab-7b46b6214435)